apps/docs/content/features/backup.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Backup data
+description: Learn how you can work with backups in Zerops works.
+---
+
+Zerops provides data backup for certain services. The data is stored in S3 storage.
```

Comment:
mozna bych trochu vic specifikovala, ktery services to aktualne jsou

Reply:
Doplnil jsem vetu,z e to je v detailu service.